src/main/scala/com/typesafe/sbt/packager/debian/NativePackaging.scala

```diff
@@ -43,6 +43,7 @@ trait DebianNativePackaging extends DebianPluginLike {
    * package.
    */
   private[debian] def debianNativeSettings: Seq[Setting[_]] = inConfig(Debian)(Seq(
+    debianNativeBuildOptions += "-Znone", // packages are largely JARs, which are already compressed
```

Comment:
It seems that `debian...` keys are scoped to the `Debian` config whenever possible.

https://github.com/pauldraper/sbt-native-packager/blob/pauldraper-deb-compress/src/main/scala/com/typesafe/sbt/packager/debian/DebianPlugin.scala#L170